source/lib/course-search/types.js

```diff
@@ -9,7 +9,7 @@ export type CourseType = {
 	level: number,
 	name: string,
 	pn: boolean,
-	prerequisites: boolean,
+	prerequisites: boolean | string,
```

Comment:
I think this is more accurately `false | string`? Unless you've seen `true` in which case I probably need to go fix the generator.

Reply:
done